frontend/src/views/dashboard/widgets/Activities.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Divider from '../../../components/Divider';
+import ActivityItem from './ActivityItem';
+
+const Activities = () => {
+  const activityList = [
+    '1',
+    '2',
+    '3',
+    '4',
+    '5',
+    '6',
+    '7',
+    '8',
+    '9',
+    '10',
+    '11',
+    '12',
+    '13',
+    '14',
+    '15',
+  ]
+  return (

```

Comment:
```suggestion
  ];
  
  return (

```
Add newline and semi-colon